ci/filter-tasks.js

```diff
@@ -111,15 +111,18 @@ var getTasksToBuildForPR = function() {
             sourceBranch = sourceBranch.split(':')[1];
         }
         run('git fetch origin pull/' + prId + '/head:' + sourceBranch);
-        run ('git checkout ' + sourceBranch);
+        run('git checkout master');
```

Comment:
We checkout master then another branch right after?

Reply:
Yeah, the other branch is the one we want to checkout, but if we don't check out master at some point the merge-base/diff will fail. Its kinda weird behavior by git - I'd expect you to be able to do the diff as long as you've fetched the branch, but it does actually require you to check it out. That's why I had the line in the first place.